.eslintrc.json

```diff
@@ -4,7 +4,8 @@
     "markdown"
   ],
   "env": {
-    "browser": true
+    "browser": true,
+    "jquery": true
```

Comment:
Is `jquery` env still needed since you remove the import statement?

Could be also removed from the `peerDependencies` list.

Reply:
After digging into it, I noticed that the jQuery environment is needed due to these following errors:

```sh
aleblanc@laptop931552: ~/Code/github/ovh-ux/ng-ovh-jsplumb on develop [!]
$ yarn run lint:js
yarn run v1.15.2
$ eslint --quiet --fix --format=pretty src

  src/instance/controller.js:194:7
  ✖  194:7   $ is not defined.  no-undef
  ✖  200:3   $ is not defined.  no-undef

  src/instance/directive.js:88:13
  ✖   88:13  $ is not defined.  no-undef
  ✖   89:31  $ is not defined.  no-undef

  4 errors

error Command failed with exit code 1.
```